lib/page/set-errors/set-errors.js

```diff
@@ -153,6 +159,18 @@ external.setSummaryErrors = (pageInstance, errorObjs = []) => {
 }
 
 external.setSummaryError = (pageInstance, controlInstance, errorType, error) => {
+  pageInstance.errorsSeen = pageInstance.errorsSeen || {}
```

Comment:
It's to prevent file upload required errors being registered if an error already exists.

But the scenario is for all controls:

- if a required error has been registered, don't add any other errors
- if any other error[s] have been registered, don't add a required error